evm/test/IntegrationRelayer.t.sol

```diff
@@ -172,7 +172,7 @@ contract TestEndToEndRelayer is
         nttManagerChain2.setPeer(chainId1, bytes32(uint256(uint160(address(nttManagerChain1)))), 9);
         DummyToken token2 = DummyTokenMintAndBurn(nttManagerChain2.token());
         wormholeTransceiverChain2.setIsWormholeRelayingEnabled(chainId1, true);
-        wormholeTransceiverChain2.setIsWormholeEvmChain(chainId1);
+        wormholeTransceiverChain2.setIsWormholeEvmChain(chainId1, true);
```

Comment:
We should add a test demonstrating that we can set the `IsWormholeEvmChain` to false after setting it to true.